ui/app/modifiers/utils/waypointHclMode.ts

```diff
@@ -0,0 +1,35 @@
+import codemirror from 'codemirror';
```

Comment:
Just for consistency, I’d tempted to use kebab case for this file (i.e. `utils/waypoint-hcl-mode.ts`)